README.md

```diff
@@ -724,7 +724,7 @@ export class Order extends Entity {
 Removing or updating the value of `foreignKeys` will be updated or delete or update the constraints in the db tables. If there is a reference to an object being deleted then the `DELETE` will fail. Likewise if there is a create with an invalid FK id then the `POST` will fail.
 " %}
 
-### Auto-generated id property
+### Auto-generated ids
```

Comment:
Just curious.. why are we renaming to `ids`? This section is applicable as long as `id:true`, right?

Reply:
Because I used `Auto-generated ids` in MySQL 🙈  I am good with either tho.

Reply:
either way is fine for me. It's good to be consistent as well. Thanks.